features/onbekend_waardes.feature

```diff
@@ -1,89 +1,299 @@
+    | naam                  | waarde   |
+    | nationaliteit         | 0000     |
+    | redenOpname           | 311      |
+    | datumIngangGeldigheid | 20030417 |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 999992934                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende gegevens
+    | naam                  | waarde    |
+    | burgerservicenummer   | 999992934 |
+    En bevat de persoon met burgerservicenummer '999992934' een nationaliteit met alleen de volgende gegevens
+    | redenOpname.code            | 311                                  |
+    | redenOpname.omschrijving    | Vaststelling onbekende nationaliteit |
+    | datumIngangGeldigheid.datum | 2003-04-17                           |
+    
+  Scenario: onbekend waarde voor reden opname nationaliteit
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    En de persoon met burgerservicenummer '999992934' heeft de volgende nationaliteit gegevens
+    | naam                  | waarde   |
+    | nationaliteit         | 0052     |
+    | redenOpname           | 000      |
+    | datumIngangGeldigheid | 20030417 |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 999992934                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende gegevens
+    | naam                  | waarde    |
+    | burgerservicenummer   | 999992934 |
+    En bevat de persoon met burgerservicenummer '999992934' een nationaliteit met alleen de volgende gegevens
+    | nationaliteit.code          | 0052       |
+    | nationaliteit.omschrijving  | Belgische  |
+    | datumIngangGeldigheid.datum | 2003-04-17 |
 
-    Gegeven de geboorteplaats van de ouder is niet bekend
-    Als de ouder wordt geraadpleegd
-    Dan is in het antwoord geboorte.plaats niet aanwezig
+Rule: de onbekendwaarde voor geslachtsaanduiding wordt wel geleverd
 
   Scenario: een onbekendwaarde bij geslachtsaanduiding wordt wel opgenomen in de response
-    Gegeven in de registratie heeft de geslachtsaanduiding de waarde "O" (onbekend)
-    Als de persoon wordt geraadpleegd
-    Dan heeft attribuut geslachtsaanduiding de waarde "Onbekend"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                        | waarde    |
+    | burgerservicenummer         | 999992934 |
+    | geslachtsaanduiding         | O         |
```

Comment:
waardetabel geslachtaanduiding ontbreekt in gegeven

Reply:
ik maak er dit van:
```
Dan bevat de persoon met burgerservicenummer '999992934' de volgende gegevens

    | naam                | waarde    |
    | burgerservicenummer | 999992934 |
    | geslachtsaanduiding | O         |
```